.circleci/config.yml

```diff
@@ -168,22 +168,22 @@ workflows:
           context: api-assume-role-housing-development-context
           requires:
             - build-and-test
-          filters:
-            branches:
-              only: master
+          # filters:
+            # branches:
+              # only: master
       - terraform-init-and-apply-to-development:
           requires:
             - assume-role-development
-          filters:
-            branches:
-              only: master
+          # filters:
+            # branches:
+              # only: master
       - deploy-to-development:
           context: api-assume-role-housing-development-context
           requires:
             - terraform-init-and-apply-to-development
-          filters:
-            branches:
-              only: master
+          # filters:
+            # branches:
+              # only: master
```

Comment:
Wouldn't this mean this steps will also run for any branches that are the release branch or based of the release branch?
Do we care about that?

Reply:
Not for dev i wouldn't have thought - I mean if release updates dev branch it should be fine.